modules/installation-osp-configuring-api-floating-ip.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+[id="installation-osp-configuring-api-floating-ip_{context}"]
+= Configuring application access with floating IP addresses
+
+After you install {product-title}, configure {rh-openstack-first} to allow application network traffic.
+
+[NOTE]
+====
+You do not need to perform this procedure if you provided values for `platform.openstack.lbFloatingIP` and `platform.openstack.ingressFloatingIP` in the `install-config.yaml` file, or `os_api_fip` and `os_ingress_fip` in the `inventory.yaml` playbook, during installation. The floating IP addresses are already set.
+====
+
+.Prerequisites
+
+* {product-title} cluster must be installed
+* Floating IP addresses are enabled as described in the {product-title} on {rh-openstack} installation documentation.
```

Comment:
Are there specific OSP CLI or access permissions that you need to complete this process?

Reply:
Shouldn't be.